articles/service-fabric/service-fabric-application-lifecycle-sfctl.md

```diff
@@ -172,6 +172,8 @@ When an application upgrade is in progress, the status can be retrieved using th
 Finally, if an upgrade is in progress and needs to be canceled, you can use
 the `sfctl application upgrade-rollback` to roll back the upgrade.
 
+NOTE - We recommend using sfctl for cluster management, howerver, for ARM, SF CLI - https://docs.microsoft.com/en-us/cli/azure/sf/cluster?view=azure-cli-latest (az sf) can still be used. 
```

Comment:
NOTE - We recommend using sfctl for cluster management, howerver, Azure Resource Manager (ARM) operations are supported in the SF CLI - https://docs.microsoft.com/en-us/cli/azure/sf/cluster?view=azure-cli-latest (az sf).

Reply:
Please fix this to be a proper markdown link. It's also confusing since the SF CLI is sfctl. Please use the Azure CLI when referring to `az sf` commands.